spec/jobs/searchgov_url_fetcher_job.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe SearchgovUrlFetcherJob do
+  subject(:perform) { SearchgovUrlFetcherJob.perform_now(args[:searchgov_url]) }
+  let!(:searchgov_url) { SearchgovUrl.create(url: 'https://agency.gov/') }
```

Comment:
are you sure you need the bang method here?

Reply:
bump

Reply:
I think I do, as the tests don't seem to pass if I don't have it.  Maybe there's a better way of organizing it?